plugins/rabbitmq_logs.yaml

```diff
@@ -0,0 +1,34 @@
+version: 0.0.1
+title: RabbitMQ
+description: Log parser for RabbitMQ
+parameters:
+  - name: daemon_log_path
+    type: string
+    default: "/var/log/rabbitmq/rabbit@*.log"
+  - name: start_at
+    type: string
+    supported:
+      - beginning
+      - end
+    default: end
+template: |
+  receivers:
+    filelog:
+      include: [ '{{ .daemon_log_path }}' ]
+      multiline:
+        line_start_pattern: '\d+-\d+-\d+ \d+:\d+:\d+\.\d+\+\d+:\d+'
+      start_at: {{ .start_at }}
+      attributes:
+        log_type: rabbitmq
+      operators:
+        - type: regex_parser
+          regex: '^(?P<timestamp>\d+-\d+-\d+ \d+:\d+:\d+\.\d+\+\d+:\d+) \[(?P<severity>\w+)\] \<(?P<process_id>\d+\.\d+\.\d+)\> (?P<message>.*)'
+          timestamp:
+            parse_from: body.timestamp
+            layout: '%Y-%m-%d %H:%M:%S.%s%j'
+          severity:
+            parse_from: body.severity
```

Comment:
Does this cover all the severity mappings? There are some differences between base stanza's default severity map and otels default severity map.

Reply:
For reference, here's what we have in the update-fields branch on stanza-plugins: https://github.com/observIQ/stanza-plugins/blob/5e885ea0023ef1df71194f63fee24d74d085f525/plugins/rabbitmq.yaml#L46

Reply:
Good call out I didn't know we had that branch fixing up for that.

Reply:
@BinaryFissionGames @jsirianni any insight on the severity. I tested this out using the install scripts we had for ops-agent and the severity does not map like we have in in the severity parsing. 

Looks like the [ops-agent parsing](https://github.com/GoogleCloudPlatform/ops-agent/blob/master/apps/rabbitmq.go#L49-L52) we did also just did normal mapping. I'm thinking of changing this back any concerns with that?

Reply:
Actually looks like the regex on that branch doesn't work at all. Taking another look.

Reply:
Looks like the timestamp doesn't work either. I'm reverting back to what I had before as that does look like it works and matches ops-agent.

Reply:
Yeah, I would just use something similar to what's in the ops agent.

Just to explain/expand on the differences a bit and what's in that file; The mapping there are basically just to make up the difference with stanza (I believe these mappings get applied over the defaults? I forget the specifics) - so a lot of these mappings on that branch are probably unnecessary, and were just applied as a catch all so we didn't have to audit each specific technology.

What we have in the ops-agent would actually work just fine with the default, if we wanted those mappings ("noti" -> "default" seems weird to me, though, that seems more like it should be warn).